### ISSUE TITLE
tex command

### ISSUE THREAD

**Issue by JosephineYu** [Macaulay2/M2#995]
The "tex" command is not putting braces { } where it should.

i34 : I = ideal(x^11);

o34 : Ideal of R

i35 : tex entries gens gb I

o35 = ${{x^11}}$

It should be x^{11} instead of x^11.

However, the following is correct:

i36 : tex gens gb I

o36 = $\bgroup\begin{pmatrix}x^{11}\\
      \end{pmatrix}\egroup$

**Comment by DanGrayson** [comment]
I can't reproduce your result:
```
Macaulay2, version 1.14
with packages: ConwayPolynomials, Elimination, IntegralClosure, InverseSystems, LLLBases,
               PrimaryDecomposition, ReesAlgebra, TangentCone, Truncations

i2 : beginDocumentation()

i3 : QQ[x]

o3 = QQ[x]

o3 : PolynomialRing

i5 : I = ideal x^11

            11
o5 = ideal x

o5 : Ideal of QQ[x]

i6 : tex entries gens gb I

o6 = $\left\{\left\{x^{11}\right\}\right\}$
```

Try running "M2 -q" instead, to see if that makes a difference.

**Comment by JosephineYu** [comment]
Hi Dan,
I was using version 1.12, so it looks like this is no longer an issue in 1.14.  Thanks!
(I’ve been having problem installing 1.14, but that is a different issue, and I’ll try to deal with it separately.)
Best,
Josephine

**Comment by DanGrayson** [comment]
Apparently it was fixed in 1.13:
```
+ /Applications/Macaulay2-1.12/bin/M2 --no-readline -q --print-width 185
Macaulay2, version 1.12
with packages: ConwayPolynomials, Elimination, IntegralClosure, InverseSystems, LLLBases, PrimaryDecomposition, ReesAlgebra, TangentCone

i1 : beginDocumentation()

i2 : QQ[x];

i3 : tex entries gens gb ideal x^11

o3 = ${{x^11}}$

i4 : 

Process M2 finished

+ /Applications/Macaulay2-1.13/bin/M2 --no-readline -q --print-width 185
Macaulay2, version 1.13
with packages: ConwayPolynomials, Elimination, IntegralClosure, InverseSystems, LLLBases, PrimaryDecomposition, ReesAlgebra, TangentCone, Truncations

i1 : beginDocumentation()

i2 : QQ[x];

i3 : tex entries gens gb ideal x^11

o3 = $\left\{\left\{x^{11}\right\}\right\}$
```
